src/tests/system/tests/test_ad.py

```diff
@@ -0,0 +1,54 @@
+"""
+SSSD Active Directory (AD) Test Cases
+
+:requirement: ad
+"""
+
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.ad import AD
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.topology(KnownTopology.AD)
+@pytest.mark.ticket(gh=7174)
+@pytest.mark.parametrize("method", ["su", "ssh"])
+@pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
```

Comment:
Do we really need to test this both under root/non-root?  I would not parametrize this.

Reply:
No, will remove it them.